packages/import-bundle/src/index.js

```diff
@@ -52,6 +53,7 @@ export async function importBundle(bundle, options = {}) {
     const { namespace } = await archive['import']({
       globals: endowments,
       __shimTransforms__: transforms,
+      // @ts-expect-error TS2740 missing properties from type
```

Comment:
I’m curious why `Compartment` is not more specifically typed here, since it should be in the `/// <reference types="ses"/>` above.

Reply:
The error message without it:
> Type 'typeof Compartment' is missing the following properties from type 'Compartment': globalThis, evaluate, import, load, and 2 more.